### DISCUSSION TITLE
Section 5: statefulFuzz_testInvariantBreaksHandler takes too long to run

### DISCUSSION THREAD

**Discussion by vascofcd** [Cyfrin/security-and-auditing-full-course-s23#191] [Q&A]
Hi everyone,

I'm encountering an issue with my `statefulFuzz_testInvariantBreaksHandler` test in Section 5:TSwap, lesson 16. It takes an unusually long time to run, even though my code is identical to Patrick's also played around with toml variables.

With these settings it takes like 3-4 minutes:

```
[fuzz]
runs = 1000
# seed = '0x2'

[invariant]
runs = 64
depth = 32
```

Could this be related to my PC's performance or configuration? Any insights or suggestions would be greatly appreciated!

Thanks in advance!

**Comment by EngrPips** [comment]
Hello @vascofcd, I think this is related to PC performance. Speed varies from machine to machine.
